BaseUI/src/components/ProfileImage.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {View, Image, TouchableOpacity, ImageBackground} from 'react-native';
+import ProfileImageProps from '../interfaces/ProfileImageProps';
```

Comment:
Props and State interfaces can be in this file. It is related to all components and screens.
If a file is too big, it can be split into several files but it's better to put it near (in the ProfileImage folder).